blueprint/index-draft

```diff
@@ -0,0 +1,242 @@
+---
+title: Send Web-Based Surveys via SMS
+author: dave.gussin
```

Comment:
Is there a way to have 2 authors here? Dave Gussin wrote the original write-up, and I adapted it for the terraform approach, so about half of this is his work. I made him the author for now.

Reply:
@diemenschmaschine92 No, one author only. You'll have to pick one person. If you think it's gonna be awesome, put your own name on it. If you don't want to be blamed, give someone else all the credit. ;) (I kid, but you really do have to pick one)